### DISCUSSION TITLE
Why did you start using gulp?

### DISCUSSION THREAD

**Discussion by ZhangChengLin** [ColorlibHQ/AdminLTE#3709] [Q&A] [answered]
Seeing yesterday’s update, Gulp has been introduced.
Why do you want to do this? Will v4 be fully transferred to Gulp? Or just increase the use of Gulp?

**Comment by danny007in** [comment, marked as answer]
speed increased
- css-compile / Saved lots of time
- lint (While developing, On save of single file, linting will start to process all files by nodemon, But Now it will lint only on changed file - while development mode) 🚀 / Saved lots of time
- html-generate easily / Saved lots of time
-  npmDist
- secure development environment

**Reply by danny007in** [reply]
css-compile became x2 faster
linting became x100 time faster

**Comment by danny007in** [comment]
> Will v4 be fully transferred to Gulp

But dist folder will be there by alpha release of v4 AdminLTE

**Reply by danny007in** [reply]
Not to worry,
There will multiple ways to install v4 AdminLTE.

like v3

Cool.

**Comment by danny007in** [comment]
> Or just increase the use of Gulp?

Only if
its
plugins are efficient
